Re-import css/css-values WPT

nt1m opened the pull request:
#### 5befb252eaaad8a2b2f89d0d6e9d654d8b123ff7
<pre>
Re-import css/css-values WPT
<a href="https://bugs.webkit.org/show_bug.cgi?id=229696">https://bugs.webkit.org/show_bug.cgi?id=229696</a>
<a href="https://rdar.apple.com/82802871">rdar://82802871</a>

Reviewed by Simon Fraser.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/1c543cd0645cf4ab41cf5053a7806e73c9bbfc76">https://github.com/web-platform-tests/wpt/commit/1c543cd0645cf4ab41cf5053a7806e73c9bbfc76</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/WEB_FEATURES.yml:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/animations/scale-interpolation-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/animations/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-IACVT-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-IACVT.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-all-types-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-all-types.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-color-invalid-cast.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-color-invalid-fallback-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-color-invalid-fallback.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-color-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-container-style-query-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-container-style-query.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-cycle-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-cycle.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-in-max.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-in-slotted-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-in-slotted-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-in-slotted.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-invalid-type-001-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-invalid-type-001.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-invalid-type-002-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-invalid-type-002.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-invalidation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-length-invalid-cast.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-length-invalid-fallback-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-length-invalid-fallback.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-length-valid-zero-nofallback.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-length-valid-zero.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-length-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-notype-fallback-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-notype-fallback-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-null-namespace-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-null-namespace.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-pseudo-elem-invalidation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-px-invalid-cast-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-px-invalid-cast.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-px-invalid-fallback-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-px-invalid-fallback.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-px-valid-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-px-valid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-security-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-security.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-universal-selector-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-universal-selector.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-complex-unresolved-serialize-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-complex-unresolved-serialize.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-infinity-nan-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-infinity-nan-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-infinity-nan-serialize-length-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-infinity-nan-serialize-length.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-invalid-parsing-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-invalid-parsing.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-numbers-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-numbers.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-serialization-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-serialization-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/calc-size-height-interpolation-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/calc-size-height-interpolation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/calc-size-interpolation-expansion-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/calc-size-interpolation-expansion.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-height-composition-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-height-composition.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-max-height-composition-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-max-height-composition.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-max-width-composition-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-max-width-composition.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-min-height-composition-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-min-height-composition.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-min-width-composition-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-min-width-composition.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-width-composition-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-width-composition.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/calc-size-flex-007-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/calc-size-flex-007.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/calc-size-flex-basis-on-column.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/calc-size-height-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/calc-size-height.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/calc-size-parsing-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/calc-size-parsing.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-unit-analysis-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-unit-analysis.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/clamp-length-serialize-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/clamp-length-serialize.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/crashtests/chrome-405422528-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/hypot-pow-sqrt-invalid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/hypot-pow-sqrt-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/hypot-pow-sqrt-serialize-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/hypot-pow-sqrt-serialize.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-conditionals-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-conditionals.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-cycle-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-cycle.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-invalidation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-invalidation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-media-invalidation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-media-invalidation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-style-invalidation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-style-invalidation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-supports-quirks-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-supports-quirks.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/max-function-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/minmax-length-percent-serialize-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/minmax-length-percent-serialize.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-computed-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-computed.tentative-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-computed.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-computed.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-invalid-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-invalid.tentative-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-invalid.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-invalid.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-serialize-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-serialize.tentative-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-serialize.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-serialize.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/random-computed.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/rcap-invalidation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/reference/vh-update-and-transition-in-subframe-iframe-ref.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/reference/vh_not_refreshing_on_chrome_iframe-ref.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/reference/vh-update-and-transition-in-subframe-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/reference/vh_not_refreshing_on_chrome-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/reference/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/rlh-on-root-lengths-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/rlh-on-root-lengths.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/round-function-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/round-function.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/signs-abs-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/signs-abs-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/support/support/README: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/support/support/swatch-green.png: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/support/support/swatch-red.png: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/support/support/w3c-import.log: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/support/vh_not_refreshing_on_chrome_iframe.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/calc-sibling-function-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/calc-sibling-function-in-shadow-dom-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/calc-sibling-function-in-shadow-dom.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/calc-sibling-function-parsing-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/calc-sibling-function-parsing.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/calc-sibling-function.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/sibling-function-container-query-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/sibling-function-container-query.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/sibling-function-descriptors.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/sibling-function-descriptors.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/sibling-function-invalidation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/sibling-function-invalidation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/sibling-function-media-query.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/sibling-function-media-query.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/tree-scoped-sibling-function-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/tree-scoped-sibling-function.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/vh-update-and-transition-in-subframe-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/vh-update-and-transition-in-subframe-iframe.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/vh-update-and-transition-in-subframe.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/vh_not_refreshing_on_chrome-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/vh_not_refreshing_on_chrome.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/w3c-import.log:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-values/rlh-on-root-lengths-expected.txt: Added.
* LayoutTests/platform/mac/TestExpectations:
* LayoutTests/tests-options.json:

Canonical link: <a href="https://commits.webkit.org/292942@main">https://commits.webkit.org/292942@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/868144be25be2f9ab48f8974fc9a5414735fe5f6

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/97512 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/17137 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/7353 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/102601 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/48041 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/17430 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/25590 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/74306 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/31488 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/100515 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/13207 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/88187 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/54651 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/12976 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/6068 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/47483 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/83005 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/6149 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/104619 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/24591 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/17941 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/83351 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/24963 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/84312 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/82772 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/20825 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/27319 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/5006 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/18185 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/24553 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/24375 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/27689 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/25949 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

weinig commented on LayoutTests/imported/w3c/web-platform-tests/css/css-values/signs-abs-computed-expected.txt:
```diff
@@ -205,4 +205,31 @@ PASS abs(-1turn) should be used-value-equivalent to 1turn
+FAIL calc(100dpi + 20dpi * sign(38px - 2em)) should be used-value-equivalent to 80dpi assert_not_equals: calc(100dpi + 20dpi * sign(38px - 2em)) isn't valid in 'image-resolution'; got the default value instead. got disallowed value undefined
+FAIL calc(3fr + 1fr * sign(42px - 2em)) should be used-value-equivalent to 4fr assert_not_equals: calc(3fr + 1fr * sign(42px - 2em)) isn't valid in 'grid-template-rows'; got the default value instead. got disallowed value "none"
+FAIL calc(3fr + 1fr * sign(40px - 2em)) should be used-value-equivalent to 3fr assert_not_equals: calc(3fr + 1fr * sign(40px - 2em)) isn't valid in 'grid-template-rows'; got the default value instead. got disallowed value "none"
+FAIL calc(3fr + 1fr * sign(38px - 2em)) should be used-value-equivalent to 2fr assert_not_equals: calc(3fr + 1fr * sign(38px - 2em)) isn't valid in 'grid-template-rows'; got the default value instead. got disallowed value "none"
```
We failed these 6 not because of calc() issues, but rather because it is using the property `image-resolution` for the 3 resolution tests and is assuming that `grid-template-rows` supports flex units, which the implementation in WebKit seemingly does not.

weinig commented on LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-invalid.tentative.html:
```diff
@@ -30,6 +30,7 @@
 test_invalid_number('progress(0,,, 10,, 200)');
 test_invalid_number('progress(0,,, 10,,, 200)');
 test_invalid_number('progress(0,, 10,, 200)');
+test_invalid_number('progress(0, 10, 200)');
```
Oh man, I just remove this case, but I guess haven't upstreamed that one yet.

nt1m replied:
I exported this here: https://github.com/web-platform-tests/wpt/commit/0801098bacff2ebf3a304c3cf4400b6fcbe2697a

nt1m committed:
Re-import css/css-values WPT
https://bugs.webkit.org/show_bug.cgi?id=229696
rdar://82802871

Reviewed by Simon Fraser.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/1c543cd0645cf4ab41cf5053a7806e73c9bbfc76

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/WEB_FEATURES.yml:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/animations/scale-interpolation-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/animations/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-IACVT-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-IACVT.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-all-types-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-all-types.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-color-invalid-cast.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-color-invalid-fallback-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-color-invalid-fallback.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-color-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-container-style-query-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-container-style-query.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-cycle-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-cycle.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-in-max.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-in-slotted-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-in-slotted-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-in-slotted.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-invalid-type-001-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-invalid-type-001.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-invalid-type-002-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-invalid-type-002.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-invalidation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-length-invalid-cast.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-length-invalid-fallback-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-length-invalid-fallback.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-length-valid-zero-nofallback.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-length-valid-zero.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-length-valid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-notype-fallback-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-notype-fallback-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-null-namespace-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-null-namespace.xhtml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-pseudo-elem-invalidation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-px-invalid-cast-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-px-invalid-cast.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-px-invalid-fallback-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-px-invalid-fallback.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-px-valid-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-px-valid.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-security-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-security.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-universal-selector-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/attr-universal-selector.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-complex-unresolved-serialize-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-complex-unresolved-serialize.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-infinity-nan-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-infinity-nan-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-infinity-nan-serialize-length-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-infinity-nan-serialize-length.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-invalid-parsing-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-invalid-parsing.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-numbers-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-numbers.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-serialization-002-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-serialization-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/WEB_FEATURES.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/calc-size-height-interpolation-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/calc-size-height-interpolation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/calc-size-interpolation-expansion-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/calc-size-interpolation-expansion.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-height-composition-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-height-composition.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-max-height-composition-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-max-height-composition.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-max-width-composition-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-max-width-composition.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-min-height-composition-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-min-height-composition.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-min-width-composition-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-min-width-composition.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-width-composition-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/interpolate-size-width-composition.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/animation/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/calc-size-flex-007-expected.xht: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/calc-size-flex-007.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/calc-size-flex-basis-on-column.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/calc-size-height-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/calc-size-height.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/calc-size-parsing-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/calc-size-parsing.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-size/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-unit-analysis-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/calc-unit-analysis.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/clamp-length-serialize-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/clamp-length-serialize.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/crashtests/chrome-405422528-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/hypot-pow-sqrt-invalid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/hypot-pow-sqrt-invalid.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/hypot-pow-sqrt-serialize-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/hypot-pow-sqrt-serialize.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-conditionals-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-conditionals.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-cycle-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-cycle.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-invalidation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-invalidation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-media-invalidation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-media-invalidation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-style-invalidation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-style-invalidation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-supports-quirks-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/if-supports-quirks.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/max-function-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/minmax-length-percent-serialize-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/minmax-length-percent-serialize.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-computed-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-computed.tentative-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-computed.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-computed.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-invalid-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-invalid.tentative-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-invalid.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-invalid.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-serialize-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-serialize.tentative-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-serialize.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/progress-serialize.tentative.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/random-computed.tentative.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/rcap-invalidation.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/reference/vh-update-and-transition-in-subframe-iframe-ref.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-values/reference/vh_not_refreshing_on_chrome_iframe-ref.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/reference/vh-update-and-transition-in-subframe-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/reference/vh_not_refreshing_on_chrome-ref.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/reference/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/rlh-on-root-lengths-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/rlh-on-root-lengths.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/round-function-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/round-function.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/signs-abs-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/signs-abs-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/support/support/README: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/support/support/swatch-green.png: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/support/support/swatch-red.png: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/support/support/w3c-import.log: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/support/vh_not_refreshing_on_chrome_iframe.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/calc-sibling-function-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/calc-sibling-function-in-shadow-dom-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/calc-sibling-function-in-shadow-dom.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/calc-sibling-function-parsing-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/calc-sibling-function-parsing.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/calc-sibling-function.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/sibling-function-container-query-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/sibling-function-container-query.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/sibling-function-descriptors.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/sibling-function-descriptors.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/sibling-function-invalidation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/sibling-function-invalidation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/sibling-function-media-query.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/sibling-function-media-query.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/tree-scoped-sibling-function-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/tree-scoped-sibling-function.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/tree-counting/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/vh-update-and-transition-in-subframe-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/vh-update-and-transition-in-subframe-iframe.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/vh-update-and-transition-in-subframe.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/vh_not_refreshing_on_chrome-expected.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/vh_not_refreshing_on_chrome.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-values/w3c-import.log:
* LayoutTests/platform/glib/TestExpectations:
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-values/rlh-on-root-lengths-expected.txt: Added.
* LayoutTests/platform/mac/TestExpectations:
* LayoutTests/tests-options.json:

Canonical link: https://commits.webkit.org/292942@main
